### PR TITLE
[4.1] fix upload multiple column

### PR DESCRIPTION
**Pull request by pxpm**
refs: #3198 

Our wrapper was preventing the column from getting the correct value.

### CODE REVIEW

File: src/resources/views/crud/columns/upload_multiple.blade.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
     @if ($value && count($value))
         @foreach ($value as $file_path)
         @php
-            $column['wrapper']['href'] = $column['wrapper']['href'] ?? ( isset($column['disk'])?asset(\Storage::disk($column['disk'])->url($file_path)):asset($column['prefix'].$file_path) );
+            $column['wrapper']['href'] = ( isset($column['disk'])?asset(\Storage::disk($column['disk'])->url($file_path)):asset($column['prefix'].$file_path) );
```
Comment (tabacitu) [inline src/resources/views/crud/columns/upload_multiple.blade.php:13]: Hmm I understand the problem, but wouldn't this prevent the `href` feature from ever being overwritten for the `upload_multiple` column type?

Maybe we can find a way to fix the bug, but keep the user ability to overwrite the `href`. What if we define the `href` above, in the `@php` block, if it wasn't already set, as a closure? I don't know...

**Review by tabacitu — CHANGES_REQUESTED** [review]

File: src/resources/views/crud/columns/upload_multiple.blade.php (hunk 1 above)
Comment (pxpm) [reply src/resources/views/crud/columns/upload_multiple.blade.php:13]: I agree with you.

Or you mean something: `$column['wrapper']['href'] = $column['wrapper']['href'] ?? function($disk, $prefix, $file_path) { return ( isset($column['disk'])?asset(\Storage::disk($column['disk'])->url($file_path)):asset($column['prefix'].$file_path) ); }`

And in this line we do `$link = is_callable($column['wrapper']['href']) ? $column['wrapper']['href']($disk, $prefix, $file_path) : $column['wrapper']['href'];` ?

Let me know what you think so I can update the PR accordingly.

Best,
Pedro

**Comment by pxpm** [comment]
Just pushed a fix that addresses your concerns. 

Hope it's fine now, from my tests it works, it will only break if developer use a wrapper that is callable and don't use the same parameters that we need, so we need to add to the docs that the wrapper for this column, if they want it to be a closure, must accepts those 3 parameters. 

Best,
Pedro

**Comment by tabacitu** [comment]
Merged into `master`. It'll be available later today as a patch update, so you can get it using `composer update`.

**Pull request merged**